src/components/scream/PostScream.js

```diff
@@ -78,18 +78,35 @@ class PostScream extends Component {
         <Dialog
           open={this.state.open}
           onClose={this.handleClose}
-          fullWidth
-          maxWidth="sm"
+          fullScreen={true}
         >
           <MyButton
             tip="Close"
             onClick={this.handleClose}
             tipClassName={classes.closeButton}
+            ml={20}
```

Comment:
what are you trying to do here? I'm pretty sure setting ml={20} doesn't do anything rn. are you trying to set margin left?